xpad.c

```diff
@@ -1896,15 +2021,27 @@ static void xpad_set_up_abs(struct input_dev *input_dev, signed short abs)
 	switch (abs) {
 	case ABS_X:
 	case ABS_Y:
+		/* GHL Strum bar */
+		if ((xpad->xtype == XTYPE_XBOXONE) && (xpad->quirks & QUIRK_GHL_XBOXONE))
+			input_set_abs_params(input_dev, abs, -32767, 32767, 0, 0);
+		break;
```

Comment:
the break should be inside the if

Reply:
Fixed break position in commit b52ecef. It is now inside the IF.